examples/resources/awscc_opensearchserverless_collection/simple_collection.tf

```diff
@@ -0,0 +1,25 @@
+# Create a Collection
+resource "awscc_opensearchserverless_collection" "simple_collection" {
+    name = "awscc-collection"
+    depends_on = [
+      awscc_opensearchserverless_security_policy.security_policy
+    ]
+}
+
+# Encryption SecurityPolicy
+resource "awscc_opensearchserverless_security_policy" "security_policy" {
+    name = "awscc-security-policy"
+    description = "created via awscc"
+    type = "encryption"
+  policy = jsonencode({
```

Comment:
careful with the indent here, its always a good practice to run `terraform fmt` on your local repo